lib/improver/utilities/cube_manipulation.py

```diff
@@ -603,32 +603,32 @@ def build_coordinate(data, long_name=None,
     Construct an iris.coord.Dim/Auxcoord using the provided options.
 
     Args:
-        data (number/list/np.array):
+        data (int  or list or numpy.ndarray):
             List or array of values to populate the coordinate points.
-        long_name (str (optional)):
+        long_name (str, optional):
```

Comment:
Apart from data these are all kwargs, can you put them under a kwargs header and remove all the "optional" bits.